README.md

```diff
@@ -2,35 +2,49 @@
 
 JSON Schema definitions for Who's On First documents
 
-## Who's On First Document types
+## JSON Schema Version
 
-At the time of writing, the [`whosonfirst-data`](https://github.com/whosonfirst-data/whosonfirst-data) repo contains two types of GeoJSON documents :-
+The schemas in this repo have been written to conform to [JSON Schema v0.6 draft](http://json-schema.org/specification-links.html#draft-6) (note to self: when _will_ JSON Schema ever reach version 1.0 and come out of draft?).
 
-1. _master_ documents; these contain the canonical definition of a Who's On First place and are named `[WOFID].geojson`.
-2. _alternate_ documents; these contain supporting definitions for a Who's On First place and are named `[WOFID]-alt-*.geojson` (currently `[WOFID]-alt-quattroshapes_pg.geojson` and `[WOFID]-alt-naturalearth.geojson` exist in the `whosonfirst-data` repo)
+## Schema Types
 
-## JSON Schema Version
+This repo contains JSON schema definitions for the following JSON file types:
 
-The schemas in this repo have been written to conform to [JSON Schema v0.6 draft](http://json-schema.org/specification-links.html#draft-6) (note to self: when _will_ JSON Schema ever reach version 1.0 and come out of draft?).
+### Who's On First Property Definitions
+
+`schema/properties/properties.json` defines the JSON schema to validate Who's On First property definition files, found in the [`whosonfirst-properties`](https://github.com/whosonfirst/whosonfirst-properties) repo.
+
+### GeoJSON Documents
+
+`schema/geojson/geojson.json` defines the JSON schema to validate a WOF specific subset of GeoJSON documents.
 
-## Schema documents
+### Who's On First Documents
 
-This repo contains the following top level schemas:
+`schema/docs/whosonfirst.json` defines the JSON schema to validate Who's On First GeoJSON documents, found in the [`whosonfirst-data`](https://github.com/whosonfirst-data/whosonfirst-data) repo and is a superset of the GeoJSON schema.
+
+## Referenced Schema Definitions
+
+In addition to the top level schemas detailed above, this repo also contains the following _referenced_ (JSON schema-speak for _included_) schema definitions:
+
+* `schema/docs/wof-properties.json` - defines a Who's on First document's `properties` object
+
+* `schema/docs/wof-hierarchy.json` - defines a Who's On First `wof:hierarchy` object
+
+* `schema/geojson/geoson-bbox.json` - defines a GeoSON `bbox` object
+* `schema/geojson/geojson-geometry.json` - defines a GeoJSON `geometry` object
+
+## Who's On First Document types
+
+At the time of writing, the [`whosonfirst-data`](https://github.com/whosonfirst-data/whosonfirst-data) repo contains two types of GeoJSON documents :-
```

Comment:
Extra `:-` chars at the end?

Reply:
That's a bit of a British stylistic thing, but now fixed; see commit `f47bc88`